liquidprompt

```diff
@@ -2971,6 +2993,48 @@ _lp_git_staged_lines() {
     lp_vcs_staged_d_lines=$lp_git_diff_shortstat_d_lines
 }
 
+# Get the remote name for the current branch.
+# NOTE: Assumes that `_lp_find_vcs` and `_lp_vcs_branch` (costly functions) have been called.
+_lp_git_remote() {
+    lp_vcs_remote=""
```

Comment:
Setting this first is good, so if you exit early it is reset.

The issue is that `_lp_git_branch()` does not do this (my bad). So switching branches will leave `$lp_vcs_branch` set, which means this function doesn't realize no branch is set.

This has never been a problem before since everything always checked the return value of `_lp_git_branch()`, but this cannot. So add a line on line 2791 to set `lp_vcs_branch=""`.